tests/test_views.py

```diff
@@ -25,6 +30,33 @@ def test_retrieve_secret(client):
     assert not secret.erased
 
 
+@pytest.mark.django_db
```

Comment:
can we add tests form form validation in here as well
we want to be sure that we can't pass the validation if we have no expiry selected with no view_once set (so forever in theory)